cmd/deploy/deploy.go

```diff
@@ -297,11 +312,19 @@ func (dc *deployCommand) runDeploy(ctx context.Context, cwd string, opts *Option
 		fmt.Sprintf("%s=%s", model.OktetoNamespaceEnvVar, okteto.Context().Namespace),
 	)
 
-	for _, command := range opts.Manifest.Deploy.Commands {
-		if err := dc.executor.Execute(command, opts.Variables); err != nil {
-			log.Infof("error executing command '%s': %s", command, err.Error())
-			return fmt.Errorf("error executing command '%s': %s", command, err.Error())
-		}
+	err = dc.deploy(opts)
+	if err != nil {
+		output = fmt.Sprintf("%s\nApp '%s' deployment failed: %s", output, opts.Name, err.Error())
+		cfg = app.SetStatus(cfg, app.ErrorStatus, output)
+	} else {
```

Comment:
could we capture `CTRL + C` to set `error` status in case the user cancels the execution of the command?

Reply:
That's working. We capture the error and update the ConfigMap in this line.
This is how the UI will show this use case
<img width="943" alt="Captura de pantalla 2022-01-20 a las 19 01 19" src="https://user-images.githubusercontent.com/25170843/150395620-85a066a6-b6e2-4e76-9434-dd5a1a0b44bf.png">

Reply:
nice!